examples/image_expansion.py

```diff
@@ -0,0 +1,104 @@
+"""
+This script illustrates several expansion methods for 2D images developed in ASPIRE package
+based on the basis functions of  Fourier Bessel (FB) and Prolate Spheroidal Wave Function (PSWF).
+"""
+
+import os
+import logging
+
+import numpy as np
+
+from aspire.basis.fb_2d import FBBasis2D
+from aspire.basis.ffb_2d import FFBBasis2D
+from aspire.basis.pswf_2d import PSWFBasis2D
+from aspire.utils.matrix import anorm
+import matplotlib.pyplot as plt
+
+logger = logging.getLogger(__name__)
+
+# Load the images from numpy array
+DATA_DIR = os.path.join(os.path.dirname(__file__), '../tests/saved_test_data')
+org_images = np.load(os.path.join(DATA_DIR, 'example_data_np_array.npy'))
+
+# Set the sizes of images 129X129
+img_size = 129
+
+#
+# Specify the normal FB basis method for expending the 2D images
+#
+fb_basis = FBBasis2D((img_size, img_size))
+# Get the expansion coefficients based on FB basis
+fb_coeffs = fb_basis.evaluate_t(org_images)
+# Reconstruct images from the expansion coefficients based on FB basis
+fb_images = fb_basis.evaluate(fb_coeffs)
+logger.info('Finish normal FB expansion and reconstruction.')
+# Calculate the maximum difference between the FB estimated images to the original images
+fb_maxdiff = np.max(abs(fb_images-org_images))
```

Comment:
Spaces around minus signs (also throughout the script).

Reply:
done.